Makefile

```diff
@@ -46,14 +46,17 @@ endif
 #----------------------------------------------------------------------
 
 # Explicitly set the default target to do everything that isn’t already done
-default: | init assets all public ;
+default: | init lint assets all public ;
 
 # Run anything that needs doing post-checkout to make this buildable
 init: node_modules ;
 
 # Use building English language as a check to see if everything works
 test: en ;
 
+lint:
+	find source -type f -name '*.json' -print -exec jsonlint -q '{}' \;
```

Comment:
> does `jsonlint` not say what it's working on anyway?

It doesn't and even worse, it doesn't report the filename along with errors :'(

Re: `-execdir`, I tried that on my machine and `find` gave me some error having to do with my personal `$PATH`. AFAICT from `find(1)` the problems with `-exec` aren't a big deal here, so I just didn't bother to debug and just took the easier path. Happy to be told otherwise though.

> EDIT: strike that, `jsonlint` only takes one filename anyway it seems

Yep :/